qemu/tests/cfg/netkvm_params_check.cfg

```diff
@@ -0,0 +1,15 @@
+- netkvm_params_check:
+    virt_test_type = qemu
+    type = netkvm_params_check
+    only Windows
+    only virtio_net
+    vhost = on
+    timeout = 360
+    smp ~= ${vcpu_maxcpus}
+    queues = ${smp}
+    netkvmco_name = "TXcapacity"
+    netkvmco_value = "64"
+    expected_log_msg = "Limit m_SGTableCapacity by 64"
+    cdroms += " virtio"
+    required_virtio_win_prewhql = [0.1.242 , )
+    required_virtio_win = [1.9.36.0, )
```

Comment:
I don't really find anywhere handing this `required_virtio_win` parameter. @leidwang Could help check? We probably need to add it to env_process.py at avocado-vt similiar as `required_qemu`, and it needs to be considered that these two restriction should be `OR` instead of `AND`.

Reply:
I added these 2 parameters last year, you can check this MR.
https://github.com/avocado-framework/avocado-vt/pull/3808/files

Reply:
Look at Leidong's code. I believe it is the `OR` restriction cause I saw the following code:
```
    # Checking required virtio-win version, if not satisfied, cancel test
    if params.get("required_virtio_win") or params.get("required_virtio_win_prewhql"):
```
https://github.com/avocado-framework/avocado-vt/pull/3808/files#diff-cec329c36adced0273f3dd816c223e259216790434676f8129ae7cd639dc8672R1259

Reply:
@leidwang @heywji Right, sorry, it was even me merging it...I need to update my local repo.